core/domain/android_validation_constants.py

```diff
@@ -32,6 +32,12 @@
 INVALID_RTE_COMPONENTS = [
     'collapsible', 'math', 'skillreview', 'tabs', 'video']
 
-MAX_CHARS_IN_TOPIC_NAME = 35
+# If any of the following values are changed, edit the corresponding value in
+# app.constants.ts as well.
+MAX_CHARS_IN_TOPIC_NAME = 39
 MAX_CHARS_IN_ABBREV_TOPIC_NAME = 12
 MAX_CHARS_IN_TOPIC_DESCRIPTION = 240
+MAX_CHARS_IN_SUBTOPIC_TITLE = 64
+MAX_CHARS_IN_SKILL_DESCRIPTION = 64
+MAX_CHARS_IN_STORY_TITLE = 39
```

Comment:
For this and chapter title, do we also need to have abbreviated versions? These lengths won’t fit on all device + font zoom configurations, so we’ll need to wrap and/or ellipsize them. Is that fine?

Reply:
Hi Ben, I think @mschanteltc gave us these lengths to use. Could you check with her?

Reply:
Yep, thread is out and I'm just waiting for her reply. We can probably live with wrapping & truncation which would immediately unblock this PR, but abbreviated titles might be better long-term especially since it has consistency with topics.

Reply:
We will not have abbreviated names, so all longer names will be ellipsized.

Reply:
As discussed via the thread, we need to solve this issue in a way that doesn't require abbreviated names. Based on that, I'm fine with approving this PR. Thanks Chantel!